### PR TITLE
Nicer login

### PR DESCRIPTION
**Pull request by ericnewcomer**

### CODE REVIEW

File: static/js/temba.js [hunk 1]
```diff
@@ -494,3 +494,20 @@ document.addEventListener('DOMContentLoaded', function () {
             });
         });
 });
+
+var setInnerHTML = function(ele, html) {
```
Comment (ericnewcomer) [inline static/js/temba.js:498]: This is a replacement for jquery eval

File: templates/contacts/contact_list_spa.haml [hunk 2]
```diff
@@ -17,6 +17,7 @@
 
 
 -block spa-title
+  .page-title
```
Comment (ericnewcomer) [inline templates/contacts/contact_list_spa.haml:20]: required for how labels work in the old world

File: templates/flows/flow_editor.haml [hunk 3]
```diff
@@ -185,9 +185,9 @@
     %script(type="text/javascript" src="{{script}}")
 
   :javascript
-    const base = '/flow/assets/{{object.org.id}}/' + new Date().getTime() + '/';
-    const api = '/api/v2/';
-    const flowType = '{{ object.engine_type }}';
+    var base = '/flow/assets/{{object.org.id}}/' + new Date().getTime() + '/';
+    var api = '/api/v2/';
+    var flowType = '{{ object.engine_type }}';
```
Comment (ericnewcomer) [inline templates/flows/flow_editor.haml:190]: We don't use `const` in non-transpiled js

File: templates/flows/flow_editor.haml [hunk 4]
```diff
@@ -273,7 +273,10 @@
       }
     };
 
-    showFlowEditor(document.getElementById('rp-flow-editor'), config);
+    // wait for our component to load, then show the editor
+    document.addEventListener("temba-floweditor-loaded", function(){
+      window.showFlowEditor(document.getElementById("rp-flow-editor"), config);
+    });
```
Comment (ericnewcomer) [inline templates/flows/flow_editor.haml:279]: Fix for race condition on editor launch

File: templates/orgs/user_two_factor_enable_spa.haml [hunk 5]
```diff
@@ -17,16 +16,13 @@
...
-
-
   
 -block form-buttons
   .mt-6.text-right
     %input.button-primary(type="submit" value='{{ submit_button_name }}')
 
 -block extra-script
   {{ block.super }}
-  %script(src="{{ STATIC_URL }}js/qrious.js")
```
Comment (ericnewcomer) [inline templates/orgs/user_two_factor_enable_spa.haml:29]: moved to top-level script block

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**